src/content-helper/dashboard-page/pages/traffic-boost/preview/preview.scss

```diff
@@ -0,0 +1,257 @@
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: var(--grid-unit-5);
+		flex: 1 0 0;
+		min-width: 0;
+
+		.traffic-boost-preview-info-title {
+			overflow: hidden;
+			color: var(--gray-900);
+			text-overflow: ellipsis;
+			white-space: nowrap;
+			font-size: var(--font-size--large);
+			font-weight: 500;
+			line-height: var(--font-size--extra-large);
+			width: 100%;
+		}
+
+		.traffic-boost-preview-info-links {
+			display: flex;
+			align-items: center;
+			align-self: stretch;
+			gap: var(--grid-unit-10);
+			font-size: var(--font-size--medium);
+			font-weight: 400;
+			line-height: var(--font-size--extra-large);
+			min-height: 24px;
+
+			.traffic-boost-preview-info-links-counter {
+				display: flex;
+				align-items: center;
+				align-self: stretch;
+				gap: var(--grid-unit-10);
+
+				button {
+					padding: var(--grid-unit-5) var(--grid-unit-10);
+					height: to_rem(22px);
+
+					&.is-pressed,
+					&:focus {
+						box-shadow: 0 0 0 1px var(--black);
+						outline: none;
+					}
+
+					&:not(.is-pressed) {
+						&:focus {
+							box-shadow: inset 0 0 0 1px var(--wp-admin-theme-color);
+							color: var(--wp-admin-theme-color);
+						}
+					}
+				}
+			}
+		}
+	}
+}
+
+.wp-parsely-traffic-boost-preview-settings-popover {
+	width: to_rem(275px);
+	margin-top: var(--grid-unit-20);
+
+	&-content {
+		width: to_rem(275px);
+		padding: var(--grid-unit-20);
+		min-width: to_rem(220px);
+	}
+}
+
+.traffic-boost-preview-footer {
+	display: flex;
+	padding: var(--grid-unit-10) var(--grid-unit-20);
+	justify-content: space-between;
+	align-items: center;
+	border-top: 1px solid var(--gray-400);
+	background: var(--white);
+
+	.traffic-boost-preview-footer-previous,
+	.traffic-boost-preview-footer-next {
+		min-width: to_rem(36px);
+	}
+
+	.traffic-boost-preview-footer-actions {
+		display: flex;
+		gap: var(--grid-unit-10);
+		align-items: center;
+
+		.components-button {
+			height: to_rem(36px);
+			padding: var(--grid-unit-10) var(--grid-unit-20);
+		}
+
+		.traffic-boost-preview-footer-navigation {
+			display: flex;
+			align-items: center;
+			gap: var(--grid-unit-5);
+
+			font-size: var(--font-size--small);
+			font-weight: 500;
+			line-height: var(--font-size--medium);
+			text-transform: uppercase;
+
+			select {
+				border: none;
+				margin-bottom: 2px;
+				font-size: var(--font-size--small);
+				font-weight: 500;
+				line-height: var(--font-size--medium);
+			}
+		}
+	}
+}
+
+// Add styles for the selection wrapper in the iframe
+.parsely-traffic-boost-selected {
+	background-color: rgba(255, 241, 184, 0.3);
+	display: inline;
+	position: relative;
+}
+
+.wp-parsely-preview-wrapper ::selection {
+	background-color: rgba(255, 0, 0, 0.2) !important;
+	color: inherit;
+}
+
+.parsely-traffic-boost-highlight {
+	position: absolute;
+	z-index: 9;
+	pointer-events: none;
+}
+
+.parsely-traffic-boost-iframe-popover {
+	background: var(--sidebar-white);
+	border-radius: to_rem(4px);
```

Comment:
I see places where we've used `to_rem()` for `4px`, and others where we didn't. I can't recall if we started using it in 3 or 4 pixels.